internal/srv/app_test.go

```diff
@@ -0,0 +1,32 @@
+package srv
+
+import (
+	"testing"
+	"testing/fstest"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetHelmValues(t *testing.T) {
```

Comment:
nit: Would prefer to see these with test cases/names so it easier to read what. you are testing in each (and easily add more)

Reply:
I would recommend to try using `t.Run("some test", func(t *testing.T){...})` where the `...` wraps you're existing tests.

Reply:
✅